README.md

```diff
@@ -68,6 +68,17 @@ Last Updated On: 18-04-2022
 
 ## Installation
 
+### [lazy.nvim](https://github.com/folke/lazy.nvim)
+
+```lua
+return {
+  'nvim-lualine/lualine.nvim',
+  event = { "BufRead", "BufNewFile", "VimEnter" },
```

Comment:
I don't understand something here.
Why not just omit event line? The event will happen in any case, won't it?

if not:

Can BufRead be fired without VimEnter? 
Maybe VimEnter is enough here?